app/router.js

```diff
@@ -51,7 +51,11 @@ Router.map(function() {
     this.route('overview', { path: '', });
   });
 
-  this.route('newsletters', { path: '/kort-bestek', });
+  this.route('newsletters', { path: '/kort-bestek', }, function() {
+    this.route('overview', { path: '', });
```

Comment:
Was unsure what to pick since overview made more sense to me.
Do you mean a rename?
```
  this.route('newsletters', { path: '/kort-bestek', }, function() {
    this.route('index', { path: '', });
    this.route('search', { path: '/bericht-zoeken', });
  });
```

Reply:
You get the index route by default in Ember. No need to define it explicitely in `router.js`. See https://guides.emberjs.com/v3.28.0/routing/defining-your-routes/#toc_index-routes